src/core/Akka/Actor/Scheduler/DedicatedThreadScheduler.cs

```diff
@@ -21,7 +20,7 @@ namespace Akka.Actor
     /// Obsolete. Use <see cref="HashedWheelTimerScheduler"/> instead.
     /// </summary>
     [Obsolete("Replaced with HashedWheelTimerScheduler [1.1.2]")]
-    public class DedicatedThreadScheduler : SchedulerBase, IDateTimeOffsetNowTimeProvider, IDisposable
+    internal class DedicatedThreadScheduler : SchedulerBase, IDateTimeOffsetNowTimeProvider, IDisposable
```

Comment:
We can just delete this class

Reply:
Removed it in 7b8473ae5d3e8a84680dd2237c9880ee56cf21a7